src/test/scala/mesosphere/marathon/core/task/termination/impl/KillServiceActorTest.scala

```diff
@@ -340,6 +372,15 @@ class KillServiceActorTest extends AkkaUnitTest with StrictLogging {
       TestInstanceBuilder.newBuilder(appId).addTaskWithBuilder().taskForStatus(mesosState, stagedAt).build().getInstance()
     }
 
+    def mockReservedInstance(appId: PathId, stagedAt: Timestamp, mesosState: mesos.Protos.TaskState): Instance = {
```

Comment:
I would say it is not a mock, it is real instance. Do we really need this helper anyway?